develop/smart-contracts/connect-your-wallet.md

```diff
@@ -1,25 +1,25 @@
 ---
-title: Connect Your Wallet to Asset Hub
-description: Learn to connect MetaMask to the Asset Hub blockchain. Set up your wallet, switch networks, and request test tokens for development.
+title: Connect Your Wallet to Polkadot
+description: Learn to connect MetaMask to Polkadot Hub. Set up your wallet, switch networks, and request test tokens for development.
 ---
 
 # Connect Your Wallet
 
-## Connect to Asset Hub Using MetaMask
+## Connect to Polkadot Hub Using MetaMask
 
-MetaMask is a popular wallet for interacting with Ethereum-compatible chains. It allows users to connect to test networks that support Ethereum-based smart contracts. This article guides setting up MetaMask, connecting to the contracts deployed on the Ethereum-compatible test network, the Westend Asset Hub, and requesting test tokens for development purposes.
+MetaMask is a popular wallet for interacting with Ethereum-compatible chains. It allows users to connect to test networks that support Ethereum-based smart contracts. This article guides setting up MetaMask, connecting to the contracts deployed on the Ethereum-compatible test network, Westend Hub, and requesting test tokens for development purposes.
 
 It's important to emphasize that MetaMask primarily facilitates interactions with smart contracts, giving you access to various chain functionalities.
 
 ## Prerequisites
 
 To get started with MetaMask, you need to install the [MetaMask extension](https://metamask.io/download/){target=\_blank} and add it to the browser. Once you install MetaMask, you can set up a new wallet and securely store your seed phrase. This phrase is crucial for recovery in case you lose access.
 
-## Connect to the Asset Hub Westend TestNet
+## Connect to the Westend Hub TestNet
 
-To connect to the Westend Asset Hub TestNet via MetaMask, you need to follow these steps:
+To connect to the Westend Hub TestNet via MetaMask, you need to follow these steps:
```

Comment:
```suggestion
To connect to the Westend Asset Hub TestNet via MetaMask, you need to follow these steps:
```

Since the configs below are for asset hub, it seems like we should avoid confusion here by keeping it as Westend Asset Hub